### DISCUSSION TITLE
How can I change a person tagged?

### DISCUSSION THREAD

**Discussion by Skalli** [nextcloud/recognize#644] [Q&A] [answered]
I can't find out how to do this, but recognize often mixes up my brother and me. I want to select the pictures with him and correct the person, but can't figure out how.
Any ideas or clues?

**Comment by phil-lipp** [comment, marked as answer]
I'd recommend using the [Memories App](https://github.com/pulsejet/memories) for managing clusters. 
For now it is a prerequisite to have a cluster for you and your brother already present, as you cant add new/empty clusters at the moment (see https://github.com/pulsejet/memories/issues/117 and #443). If you already have clusters present for you two then it's as simple as navigating to one of them in Memories -> People -> YourName/YourBrothersName, selecting the photos you wanna move over, click the 3-dot-menu in the top right, select "Move to another person" and in the pop-up select the destination cluster.
